src/python/pants/backend/project_info/rules/dependencies.py

```diff
@@ -11,6 +12,12 @@
 from pants.engine.selectors import Get
 
 
+class DependencyType(Enum):
+  INTERNAL = "internal"
```

Comment:
Internal and external are really unclear names to me, and I didn't know what they meant until I read your helptext on the flag. How would you feel about source and 3rdparty?

Reply:
Great recommendations! Thank you!

Reply:
Great recommendations! Thank you!